scitos.hmx/scitos.hmx.core/src/test/java/org/hmx/scitos/hmx/core/ModelHandlerImplTest.java

```diff
@@ -1309,6 +1309,51 @@ public void testSplitProposition_6() throws HmxException {
     }
 
     /**
+     * Test: of {@code splitProposition(Proposition, ClauseItem)} after the first item of the first part of a {@link Proposition} with two enclosed
+     * children.
+     *
+     * @throws HmxException
+     *             impossible to split {@link Proposition} part after first of two items
+     */
+    @Test
+    public void testSplitProposition_7() throws HmxException {
```

Comment:
This tests the fix for an unrelated bug I found while testing the new features. :)